src/rgw/rgw_rest_s3.cc

```diff
@@ -3408,6 +3408,168 @@ int RGWPostObj_ObjStore_S3::get_encrypt_filter(
   return res;
 }
 
+struct S3Location {
```

Comment:
@shreyanshjain7174 .. as discussed with Kyle and Daniel in the slack, we are not doing OutputLocation as it is associated with Restore Select query.

Reply:
okay! done